bin/interpreter.sh

```diff
@@ -185,12 +189,25 @@ addJarInDirForIntp "${LOCAL_INTERPRETER_REPO}"
 
 CLASSPATH+=":${ZEPPELIN_INTP_CLASSPATH}"
 
+if [[ ! -z "$ZEPPELIN_IMPERSONATE_USER" ]]; then
+    INTERPRETER_RUN_COMMAND=${ZEPPELIN_IMPERSONATE_RUN_CMD}" '"
```

Comment:
Yes the " '" startes here (L193)  and ends at L206. I can add a space between `'` and `source` but since it was working I didn't do it.